etas/evaluation.py

```diff
@@ -0,0 +1,438 @@
+import sys
+from etas.inversion import ETASParameterCalculation, read_shape_coords, polygon_surface, round_half_up, parameter_dict2array, haversine
+from etas.simulation import simulate_aftershock_time
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Polygon
+from joblib import Parallel, delayed
+from tabulate import tabulate
+import numpy as np
+from scipy.special import expi
+import json
+from scipy.integrate  import quad
+from shapely.geometry import Polygon, Point, LineString
+import matplotlib.pyplot as plt
+
+
+def compute_dist_squared_from_i(i,lat_rads: np.ndarray,long_rads: np.ndarray,earth_radius=6.3781e3):
+
+    return np.square(
+                haversine(
+                    lat_rads[i],
+                    lat_rads[:i],
+                    long_rads[i],
+                    long_rads[:i],
+                    earth_radius,
+                )
+            )
+
+def to_days(timediff):
+    return timediff / np.timedelta64(1, 'D')
+
+
+
+class ETASLikelihoodCalculation(ETASParameterCalculation):
+    def __init__(self, metadata: dict):
+        """
+        Class to invert ETAS parameters.
```

Comment:
shouldn't this be the docstring describing the new class? it's ok not to do it now, but we can adjust it at some point